src/boot/loader.js

```diff
@@ -1,65 +1,41 @@
 // This file holds the logic to generate a code-splitting loader function.
 // The code is compiled on build-time.
-var modules = {
+const modules = {
   // >> boot/modes
   // app
   //   The main game application. This will bring up the Title Screen.
-  app: 'bemuse/app',
+  app: import(/* webpackChunkName: 'app' */ 'bemuse/app'),
```

Comment:
`import()` is eager: The chunk starts loading when `import()` is called, not when the promise is `.then()`. This causes all modes to be downloaded right away:

![image](https://user-images.githubusercontent.com/193136/34652251-508727d8-f40e-11e7-93d7-40a33e8d66c8.png)

To fix this, we can put the `import()` call into an arrow function:

```js
  app: () => import(/* webpackChunkName: 'app' */ 'bemuse/app'),
```

Reply:
Got it, fixing this before I go to bed as well. :ok_hand: